awesome_dashboard/static/src/dashboard/pieChart/PieChartCard.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates xml:space="preserve">
+    <t t-name="awesome_dashboard.PieChartCard" owl="1">
```

Comment:
I believe the `owl="1"` attribute does not exist anymore. It was there in early versions of Owl where Owl and non-Owl templates had to coexist.